Resources/doc/reference/amazon_s3.rst

```diff
@@ -3,13 +3,17 @@ Amazon S3
 
 In order to use Amazon S3, you will need to add the following dependency to your composer.json:
 
+.. code-block:: bash
+
     composer require aws/aws-sdk-php
 
 Configuration
 -------------
 
 This is a sample config file to enable amazon S3 as a filesystem & provider:
 
+.. code-block:: yaml
```

Comment:
travis is failing, maybe you need to add a configuration block before and fix the indetion like this:

```
.. configuration-block::

    .. code-block:: yaml

        foo
```